src/backroom/set_blinks_saccades_to_nan.m

```diff
@@ -28,7 +28,7 @@
     %   - all elements in right data columns (except blink/saccade) that correspond to right
     %     blink or saccade rows are set to NaN
     if nargin == 3
-        fn_is_left = @(x) endsWith(x, '_l');
+        fn_is_left = @(x) strcmp(x(end-1:end), '_l');
```

Comment:
Is there a reason for different endswith strategies?
this is index based, and then here for example is regex based
```
~isempty(regexp(msgline,'sec$'))
```
I'd assume the index is faster

Reply:
Yes there is a reason, regexp allows to use a cell array and the index based method not. So I've used the latter one in situations when there was only one string and regexp when the variable could be an array of strings. I could have chosen regexp for all `endsWith` but it was easier to use the index method :sweat_smile: 

Do you prefer to have regexp everywhere ?

Reply:
ok great, makes sense, no I think that what you've done helps to make the distinction